### PR TITLE
Feat: article flags

### PR DESCRIPTION
**Pull request by sericaia**
https://nidigitalsolutions.jira.com/browse/REPLAT-324

### CODE REVIEW

**Comment by trevorah** [comment]
Fonts arent loading for me...

screen shot 2017-07-27 at 13 57 25

File: packages/article-flag/article-flag.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    color: "#4D4D4D"
+  },
+  updated: {
+    color: "#3C81BE"
+  }
+};
+
+const getStyles = StyleSheet.create(styles);
+
+const ArticleFlag = ({ title, value, style }) => {
```
Comment (trevorah) [inline packages/article-flag/article-flag.js]: Whats `value` used for?

Comment (trevorah) [inline packages/article-flag/article-flag.js]: Also, how about just accepting `color` rather than the whole of `style`.

Id also rather have the interface as:
```js
import ArticleFlag, { NewArticleFlag, SponsoredArticleFlag, UpdatedArticleFlag } from "@times-components/article-flag";
```

where

```
const ArticleFlag = ({ text, color }) => ...something

const NewArticleFlag = () => <ArticleFlag text=new color=#E34605 />
const SponsoredArticleFlag = () => <ArticleFlag text=sponsored color=#4D4D4D />
const UpdatedArticleFlag = () => <ArticleFlag text=updated color=#3C81BE />
```

**Comment by sericaia** [comment]
out

File: packages/article-flag/article-flag.js (hunk 1 above)
Comment (sericaia) [inline packages/article-flag/article-flag.js]: Fixing it! 👍

File: packages/article-flag/article-flag.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      color
+    }
+  });
+
+const ArticleFlag = ({ title, color }) => {
+  if (!title) {
+    return null;
+  }
+
+  const styles = getStyles(color);
```
Comment (trevorah) [inline packages/article-flag/article-flag.js]: 👎  You are creating the stylesheet on the fly, which will affect performance.

This should fix it:
```
<Text style={[styles.title, { color }]}>{title.toUpperCase()}</Text>
```

**Comment by sericaia** [comment]
Styles fixed on android:
image

File: packages/article/package.json [hunk 3]
```diff
@@ -49,6 +49,7 @@
   },
   "dependencies": {
     "@times-components/ad": "0.0.4",
+    "@times-components/article-flag": "0.0.1",
```
Comment (trevorah) [inline packages/article/package.json]: do this in another pr
